lib/rad_common/engine.rb

```diff
@@ -10,7 +10,6 @@ class Engine < ::Rails::Engine
     require 'authtrail'
     require 'csv'
     require 'devise'
-    require 'devise-authy'
```

Comment:
if you haven't already, you need to replace this line with 
`require 'devise-twilio-verify'